pkg/server/plugin/upstreamauthority/vault/vault_test.go

```diff
@@ -88,29 +89,33 @@ func (vps *VaultPluginSuite) Test_Configure() {
 			name:       "Configure plugin with AppRole authentication params given as environment variables",
 			configTmpl: testAppRoleAuthConfigWithEnvTpl,
 			envKeyVal: map[string]string{
-				"VAULT_APPROLE_ID":        "test-approle-id",
+				"VAULT_APPROLE_ID":        "test-approle-idd",
```

Comment:
typo?

Reply:
oops, a left over from a test. Thanks for caching it!